controller/cache/cluster.go

```diff
@@ -90,7 +90,7 @@ func createObjInfo(un *unstructured.Unstructured, appInstanceLabel string) *node
 			APIVersion: "",
 		})
 	}
-	info, networkingInfo := getNodeInfo(un)
+	info, images, networkingInfo := getNodeInfo(un)
```

Comment:
Looks like `getNodeInfo` returns too many values now. Can you please change it to  `setNodeInfo(node *nodeInfo)` (and same for `setPodInfo` etc)?